source/Irrlicht/CSceneNodeAnimatorCameraModifiedMaya.cpp

```diff
@@ -238,6 +241,15 @@ namespace irr
 			}
 
 			// Rotation ------------------------------------
+
+			auto getValueDependentOnHandOrientation = [&](const float& expression)
```

Comment:
in reference to https://github.com/buildaworldnet/IrrlichtBAW/pull/378/files#r336174449